root/etc/e-smith/events/actions/nethserver-directory-user-delete

```diff
@@ -36,8 +36,14 @@ if ($userName =~ /@/) {
     $userName = (split(/@/,$userName))[0];
 }
 
-# this also removes user primary group
-my $discard = `/usr/sbin/luserdel -r '$userName'`;
+my ($name,$passwd,$uid,$gid, $quota,$comment,$gcos,$home,$shell,$expire) = getpwnam($userName);
```

Comment:
To get (N-1)th field of password DB, see for instance

https://github.com/NethServer/nethserver-mail-server/blob/92a12d0b7b0bc54c444037706ca293e383a133a4/root/etc/e-smith/templates/etc/pam.d/dovecot-master/10base#L5

A more readable solution is `use User::pwent`

https://github.com/NethServer/nethserver-directory/blob/ada5ffa24a0b55cf555169a78ad988dc5aa4caec/root/etc/e-smith/events/actions/nethserver-directory-migrate#L95